fetch/src/5_log.r

```diff
@@ -0,0 +1,19 @@
+log_data <- function() {
```

Comment:
I think we can also group this into the `2_process` phase, and can read in the saved `model_summary_results.csv` as the object `eval_data`